webhook-bridge/svix-webhook-bridge.example3.yaml

```diff
@@ -0,0 +1,74 @@
+log_level: "debug"
+log_format: "json"
+
+# Optionally configure these settings to export OTEL data
+opentelemetry:
+  address: "http://localhost:1234"
+  service_name: "my-bridge"
+  sample_ratio: 0.5
+
+# Host/port to listen on when receiving webhooks
+receiver_http_addr: "0.0.0.0:5000"
+
+# Global svix settings.
+# A svix client is needed by each sender.
+# Each sender can override any of these fields by defining a `svix_client` section.
+global_svix_client:
+  token: "XXXXX"
+  options:
+    debug: true
+    server_url: http://example.com/
+
+# Webhook Senders configure an input and (optional) transformation.
+# Use transformations to conform the payload we get as input to the expected shape (app_id, message, etc).
+# When transformation is left unset, the payload is used as-is.
+senders:
+  - name: "rabbit-local"
```

Comment:
I think name is probably a good idea, so maybe have it for the receivers too?

Reply:
Seemed like the path would be useful in that case, but I could also see the use for a name.

My main want for this tbh is to include it in spans/logging.